pkg/volume/local/local.go

```diff
@@ -204,8 +221,25 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
-		return nil
+	refs, err := mount.GetMountRefsByDev(m.mounter, m.globalPath)
+	if fsGroup != nil {
```

Comment:
Can we check fsGroup nil first, and then call GetMountRefsByDev?  Because GetMountRefs is expensive.